pkg/daemon/update.go

```diff
@@ -1494,6 +1499,39 @@ func (dn *Daemon) presetUnit(unit ign3types.Unit) error {
 	return nil
 }
 
+// writeMissingAuthorizedKeys: In OKD /var/home/core/.ssh/authorized_keys is not created during ignition.  Create
+// from /var/home/core/.ssh/authorized_keys.d/ignition if missing
+func (dn *Daemon) writeMissingAuthorizedKeys() error {
+	authKeyPath := filepath.Join(coreUserSSHPath, "authorized_keys")
+	authKeyFragPath := filepath.Join(coreUserSSHPath, "authorized_keys.d", "ignition")
+
+	// if authKeyFragPath doesn't exist return as the rest is unneeded
```

Comment:
so this covers OCP & OKD if there are no sshkeys?

Reply:
I believe so...  I can't test OCP though...

Reply:
The `core` user is created by Ignition by default unless the user opts out.

I worried a bit by two assumptions:
- not checking to see if this is a CoreOS variant (i.e. not RHEL)
- the assumption that the user exists and that user's path. The assumption of the user's path could be problematic. 

Perhaps something like:

```
	if !dn.os.IsCoreOSVariant() {
		return nil
	}

	user, err := user.Lookup("core")
	if err != nil {
		return nil
	}

	authKeyPath := filepath.Join(user.HomeDir, "authorized_keys")
	authKeyFragPath := filepath.Join(user.HomeDir, "authorized_keys.d", "ignition")

       <do your thing...>
```